*** CMakeLists.txt.orig	2004-05-26 11:03:17.000000000 +0200
--- CMakeLists.txt	2004-05-26 11:03:58.000000000 +0200
***************
*** 22,30 ****
  vtkPolyDataSourceWidget.cxx
  vtkPolyDataToImageStencil.cxx
  vtkProcrustesAlignmentFilter.cxx
! vtkPushImageFilterSample
! vtkPushImageReader
! vtkPushPipeline
  vtkRIBExporter.cxx
  vtkRIBLight.cxx
  vtkRIBProperty.cxx
--- 22,30 ----
  vtkPolyDataSourceWidget.cxx
  vtkPolyDataToImageStencil.cxx
  vtkProcrustesAlignmentFilter.cxx
! vtkPushImageFilterSample.cxx
! vtkPushImageReader.cxx
! vtkPushPipeline.cxx
  vtkRIBExporter.cxx
  vtkRIBLight.cxx
  vtkRIBProperty.cxx
